src/screens/Heading.js

```diff
@@ -18,7 +18,13 @@ export default () => (
       syntaxes={{
         ...genericSyntaxes,
         a11yTitle: 'Open Drop',
-        color: 'neutral-1',
+        color: [
+          'neutral-1',
+          {
+            dark: 'dark-1',
```

Comment:
I think the example colors should be reversed. If the background color is 'dark', the text color should be light.